tests/cli/test_cli.py

```diff
@@ -692,11 +692,12 @@ def test_run_cron_elapsed(runner, tmp_path):
     create_example_project(tmp_path)
 
     # Create and backfill `prod` environment
-    with freeze_time("2023-01-01 23:59:00"):
+    with time_machine.travel("2023-01-01 23:59:00 UTC", tick=False) as traveler:
```

Comment:
Why was `UTC` needed in this test but not in other places?

Reply:
Based on this from the docs, I had assumed it wouldn't be needed anywhere.

After some testing, my inference is that if nothing is passed to a `datetime`/`time` method anywhere in the call stack everything will automatically be UTC.

If a tz-aware object is passed, the local timezone can come into play. When that happens, we sometimes need to manually specify the UTC tz.

```
A datetime.datetime. If it is naive, it will be assumed to have the UTC timezone. If it has tzinfo set to a [zoneinfo.ZoneInfo instance](https://docs.python.org/3/library/zoneinfo.html#zoneinfo.ZoneInfo), the current timezone will also be mocked.
```